Sources/DolbyIORTSCore/StreamOrchestrator.swift

```diff
@@ -264,6 +264,22 @@ private extension StreamOrchestrator {
         let (_, success) = await (startSubscribeStateUpdate, startSubscribe)
         return success
     }
+    
+    func stopAudio(for sourceId: String) async {
```

Comment:
suggestion: can we mark this method private ?

Reply:
It's under private extension